packages/browser/src/plugins/segmentio/__tests__/batched-dispatcher.test.ts

```diff
@@ -91,11 +91,11 @@ describe('Batching', () => {
 
     expect(fetch).toHaveBeenCalledTimes(1)
     expect(fetch.mock.calls[0]).toMatchInlineSnapshot(`
-      Array [
+      [
         "https://https://api.segment.io/b",
-        Object {
-          "body": "{\\"batch\\":[{\\"event\\":\\"first\\"},{\\"event\\":\\"second\\"},{\\"event\\":\\"third\\"}],\\"sentAt\\":\\"1993-06-09T00:00:00.000Z\\"}",
-          "headers": Object {
+        {
+          "body": "{"batch":[{"event":"first"},{"event":"second"},{"event":"third"}],"sentAt":"1993-06-09T00:00:00.000Z"}",
```

Comment:
Whoa, the `"{"batch"` was really tripping me up, but I guess with the string tag it's valid!